docs/breaking-changes.md

```diff
@@ -30,6 +30,28 @@ webContents.setWindowOpenHandler((details) => {
 })
 ```
 
+### Removed: `webContents.decrementCapturerCount(stayHidden, stayAwake)`
```

Comment:
Unless there's a compelling reason to deprecate/remove decrement separately from increment, it seems like it would be less confusing to do the whole change at once, e.g. deprecations in 22 and removed in 23.

Doing them separately seems like extra work for app devs with no real upside? (Or maybe an upside that I'm msising?)

Reply:
there's not - just forgot to add to this doc!